src/odemis/util/angleres.py

```diff
@@ -515,6 +515,7 @@ def Rectangular2Polar(data, output_size, colormap=None, hole=False):
     mask = _CreateMirrorMaskRectangular(data, hole=hole)
 
     # Get the upper/lower intensity limit, by checking for a fixed amount of outliers.
+    data[numpy.isnan(data)] = 0  # TODO small hack to handle potential nans in data (present in normalized img)
```

Comment:
There is a dedicated numpy function for that (which might be a little bit faster). 
I don't think it's a TODO, it's just a "small hack". Maybe the TODO would be to remove this hack and to compute the projection without taking the NaN into account?